django_rq/workers.py

```diff
@@ -12,28 +15,55 @@ def get_exception_handlers():
         'EXCEPTION_HANDLERS': ['path.to.handler'],
     }
     """
+    from .settings import EXCEPTION_HANDLERS
+
     return [import_attribute(path) for path in EXCEPTION_HANDLERS]
 
 
+def get_worker_class(worker_class=None):
+    """
+    Return worker class from RQ settings, otherwise return Worker.
+    If `worker_class` is not None, it is used as an override (can be
+    python import path as string).
+    """
+    RQ = getattr(settings, 'RQ', {})
+
+    if worker_class is None:
+        worker_class = Worker
+        if 'WORKER_CLASS' in RQ:
+            worker_class = RQ.get('WORKER_CLASS')
+
+    if isinstance(worker_class, six.string_types):
+        worker_class = import_attribute(worker_class)
+    return worker_class
+
+
 def get_worker(*queue_names, **kwargs):
     """
     Returns a RQ worker for all queues or specified ones.
     """
-    queues = get_queues(*queue_names)
-    name = kwargs.get('name')
-    return Worker(queues,
-                  connection=queues[0].connection,
-                  exception_handlers=get_exception_handlers() or None,
-                  name=name)
+    job_class = get_job_class(kwargs.pop('job_class', None))
```

Comment:
Can we use `kwargs.get('job_class')` instead? This already returns `None` by default and it doesn't mutate `kwargs` so downstream function calls can access `job_class` if needed.

Reply:
But all downstream function calls already get `job_class` (imported here) which will never be `None`. Also, passing `**kwargs` to `worker_class` with `job_class` will probably error out with `TypeError` (got multiple values for keyword argument job_class).